src/platform/windows/mod.rs

```diff
@@ -227,6 +227,11 @@ impl Platform for PlatformImpl {
         })))
     }
 
+
+    fn cpu_time(&self) -> io::Result<Vec<CpuTime>> {
+        todo!("implement cpu_time on windows")
```

Comment:
uhhh.. Are you planning to do that in this PR before I land it? Otherwise it should also be a `        Err(io::Error::new(io::ErrorKind::Other, "Not supported"))`

Reply:
> uhhh.. Are you planning to do that in this PR before I land it? Otherwise it should also be a `        Err(io::Error::new(io::ErrorKind::Other, "Not supported"))`

I can update it to not supported if that's fine with you the windows stuff seems complicated.